docs/boundary_conditions_cells.rst

```diff
@@ -0,0 +1,41 @@
+Boundary Conditions for Cells
+##############################
```

Comment:
This is somewhat duplicate content from potts.rst, but I think it's valuable to have smaller quick reference pages like this as well.